dts/bindings/i2c/nuvoton,npcx-i2c-port.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-I2C port pads node
+
+compatible: "nuvoton,npcx-i2c-port"
+
+include: [base.yaml]
+
+properties:
+    label:
+        required: true
+
+    port:
+        type: int
+        required: true
+        description: index of i2c port
+
+    controller:
+        type: phandle
+        required: true
+        description: i2c controller to handle signals from port pads
+
+    pinctrl:
+        type: phandles
+        required: true
+        description: configurations of pinmux controllers
+
```

Comment:
No empty lines at end of file.  (The last non-empty line should end with a newline, though.)

Reply:
Done.